kerl

```diff
@@ -782,8 +782,14 @@ maybe_patch_catalina() {
     fi
 }
 
+is_osx() {
+    uname="$(uname -r | grep "^$1.*$")"
+    [ "$uname" != "" ] && return
```

Comment:
I am not a huge fan of this construct - it's shorthand and cute but I'd prefer a more explicit and traditional if; then; else block, I guess.

Reply:
Sure. No prob. 😄

Reply:
How about 0f2ab1f? I may also change `return` to `return 0` and `false` to `return 1`, if you prefer.